pkg/controllers/work/apply_controller.go

```diff
@@ -340,6 +346,16 @@ func (r *ApplyWorkReconciler) applyUnstructured(ctx context.Context, gvr schema.
 		Name:      manifestObj.GetName(),
 		Namespace: manifestObj.GetNamespace(),
 	}
+	// Marshaling adds about 100-200 bytes to the object's actual size
+	marshaledObj, marshalErr := json.Marshal(manifestObj)
+	if marshalErr != nil {
+		return nil, ManifestNoChangeAction, marshalErr
+	}
+	if len(marshaledObj) > (TotalAnnotationSizeLimitB - len(manifestHashAnnotation) - len(lastAppliedConfigAnnotation) - manifestHashLength) {
+		klog.V(2).InfoS("Size of manifest object is greater than 262,013 bytes", "Name", manifestObj.GetName(), "size of marshaled object", len(marshaledObj), "kind", manifestObj.GetKind())
```

Comment:
I believe this log is to indicate a flow change. That being the object will be applied, but we won't be storing the manifest within the annotation. Should we indicate that within the log?

Reply:
Changed log